_config.yml

```diff
@@ -0,0 +1,31 @@
+title: Data Driven Development
+author: AICoE
+logo: logo.png
```

Comment:
where is this image located?

Reply:
ah good catch! will upload the image :+1: